pkg/time/time_linux.go

```diff
@@ -137,14 +145,20 @@ func ModifyTime(pid int, deltaSec int64, deltaNsec int64) error {
 	}
 	fakeAddr := fakeEntry.StartAddress
 
-	// 115 is the index of TV_SEC_DELTA in fakeImage
-	err = program.WriteUint64ToAddr(fakeAddr+115, uint64(deltaSec))
+	// 139 is the index of CLOCK_IDS_MASK in fakeImage
```

Comment:
The order of the index does not match the 79 comment rows?

Reply:
I should modify 77-79 comments. They should follow the order of being used in program.